ogre_mesh_viewer.py

```diff
@@ -236,6 +236,8 @@ def preRenderTargetUpdate(self, evt):
 
             if entity is not None and ImGui.BeginMenu("View"):
                 enode = entity.getParentSceneNode()
+                if ImGui.MenuItem("Show/Hide Side Panel", "H", self.app.axes_visible):
+                    self.app._toggle_side_panel()
```

Comment:
```suggestion
                if ImGui.MenuItem("Show Side Panel", None, self.side_panel_visible):
                    self.side_panel_visible = not self.side_panel_visible
```
- we dont need a hotkey for this
- wrong status variable
- use "show" as there will be a checkbox

Reply:
![image](https://github.com/OGRECave/ogre-meshviewer/assets/5248597/88334849-87c1-4c6d-af00-510b84c6da22)

Can I have hotkey, like in blender?

Reply:
but please choose another hotkey, as "H" is for help - "S" maybe? or what does blender use?

Also keep the status variable in `self.gui.side_panel_visible` - the others are in `app` because Ogre is involved.

Reply:
Blender uses "N" for the side panel.

But I would suggest to use "P" for this and "S" for "save screenshot".

What do you think?

Reply:
I would like to avoid changing key for already release versions, so no "P".

Why not go with "N" to be compatible with blender? Not all bindings must make sense ^^

Reply:
I understand, but how many people are depending on that key being "P"?

I don't want to say that no one uses `ogre-meshviewer`. 
What I'm saying is that even if someone uses it it's not like there are scripts depending on that.

And the "S" makes more sense...

It doesn't seem like there is going to be a huge backlash for changing that.

Just my opinion.

Reply:
> It doesn't seem like there is going to be a huge backlash for changing that.

probably not. And you are the current expert user, we can change that.

some random thoughts:
- "P" was used original as in "Print Screen"
- blender refers to the Sidepane as "Context", so we could also got with "C"

Reply:
OK, nevermind.

Let's use your original suggestion of "N", because "C" is even worse 😅

Reply:
No merge?